packages/googlehealthcare/configuration-schema.json

```diff
@@ -18,23 +18,42 @@
       "type": "string",
       "description": "The API version",
       "default": "v1",
-      "examples": ["v1", "v1beta1"]
+      "examples": [
+        "v1",
+        "v1beta1"
+      ]
     },
     "cloudRegion": {
       "type": "string",
-      "examples": ["us-central1"]
+      "title": "cloudRegion",
+      "description": " The dataset location",
+      "examples": [
+        "us-central1"
+      ]
     },
     "projectId": {
       "type": "string",
-      "examples": ["adjective-noun-123"]
+      "title": "projectId",
+      "description": "The ID of your Google Cloud project",
+      "examples": [
+        "adjective-noun-123"
+      ]
     },
     "datasetId": {
       "type": "string",
-      "examples": ["my-dataset"]
+      "title": "datasetId",
+      "description": "The FHIR store's parent dataset",
```

Comment:
I honestly don't know, is the word "parent" here correct?

Reply:
That is a good question, according to the docs it says `the FHIR store's parent dataset` - https://cloud.google.com/healthcare-api/docs/how-tos/fhir-resources#healthcare-create-patient-drest